src/AuthenticationServices/ASAuthorizationProviderExtensionLoginManager.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Runtime.Versioning;
+using Foundation;
+using Security;
+
+#nullable enable
+namespace AuthenticationServices {
+
+	public unsafe partial class ASAuthorizationProviderExtensionLoginManager {
+
+#if MONOMAC
```

Comment:
The class itself is only available on macOS; so the `#if MONOMAC` condition should be around the entire class.